collectors/ioping.plugin/README.md

```diff
@@ -1,86 +1,120 @@
 ```
 
-## alarms
+### Setting up multiple ioping plugins with different settings
 
-Netdata will automatically attach a few alarms for each host.
-Check the [latest versions of the ioping alarms](https://raw.githubusercontent.com/netdata/netdata/master/health/health.d/ioping.conf)
+In case you have multiple nodes that should be pinged in different intervals, you need to run multiple ioping plugins with different settings.
+Netdata allows you to add as many `ioping` plugins as you like.
+For example, you may need to ping one node once per 10 seconds, and another once per second.
 
-## Multiple ioping Plugins With Different Settings
+To set up multiple ioping plugins:
 
-You may need to run multiple ioping plugins with different settings or different end points.
-For example, you may need to ping one destination once per 10 seconds, and another once per second.
+1. Open a new terminal and change into the configuration directory.
 
-Netdata allows you to add as many `ioping` plugins as you like.
+   ```bash
+   cd /etc/netdata
+   ```
 
-Follow this procedure:
+2. Copy the original ioping configuration file. You can name the file any name you like.
 
-**1. Create New ioping Configuration File**
+   ```bash
+   cp ioping.conf ioping2.conf
+   ```
 
-```sh
-# Step Into Configuration Directory
-cd /etc/netdata
+3. Edit `ioping2.conf` and set the settings and the destination you need for the second instance.
 
-# Copy Original ioping Configuration File To New Configuration File
-cp ioping.conf ioping2.conf
-```
+4. To complete the next steps, you need change to the superuser.
 
-Edit `ioping2.conf` and set the settings and the destination you need for the seconds instance.
+   ```bash
+   sudo su
+   ```
 
-**2. Soft Link Original ioping Plugin to New Plugin File**
+5. Change into the plugins directory.
 
-```sh
-# Become root (If The Step Step Is Performed As Non-Root User)
-sudo su
+   ```bash
+   cd /usr/libexec/netdata/plugins.d
+   ```
 
-# Step Into The Plugins Directory
-cd /usr/libexec/netdata/plugins.d
+6. Create a soft link to connect the original ioping plugin to new plugin file. Make sure the plugin and the configuration file have the same name (in our example "ioping2").
 
-# Link ioping.plugin to ioping2.plugin
-ln -s ioping.plugin ioping2.plugin
-```
+   ```bash
+   ln -s ioping.plugin ioping2.plugin
+   ```
+
+   Netdata will detect the new plugin and start it.
 
-That's it. Netdata will detect the new plugin and start it.
+## Metrics and Alerts produced by this collector
 
-You can name the new plugin any name you like.
-Just make sure the plugin and the configuration file have the same name.
+| Chart  | Metrics        | Alert                                                                             |
+| ------ | -------------- | --------------------------------------------------------------------------------- |
+| ioping | ioping.latency | [ioping_disk_latency](https://community.netdata.cloud/t/ioping-disk-latency/2120) |
```

Comment:
```suggestion
| read_latency | ioping.latency | [ioping_disk_latency](https://community.netdata.cloud/t/ioping-disk-latency/2120) |
```

I am not sure what "chart" is, but "read_latency" is closer to the truth.

Reply:
> [ioping_disk_latency](https://community.netdata.cloud/t/ioping-disk-latency/2120)

I think we need 2 links:
 - [conf](https://github.com/netdata/netdata/blob/master/health/health.d/ioping.conf)
 - [troubleshoot](https://community.netdata.cloud/t/ioping-disk-latency/2120)